apps/search/style/search.css

```diff
@@ -291,3 +290,12 @@ gaia-grid {
 [data-loading=true] .loading {
   display: block;
 }
+
+html[dir="rtl"] #suggestions-provider {
+  padding: 0 1rem 0 2rem;
```

Comment:
I tried to remove the `direction: ltr;` in https://github.com/mozilla-b2g/gaia/pull/28630/files#diff-f5140311864b481688975f876b111d48L23, but I bump into a platform bug where the #suggestions-provider element reports no width when in RTL, so I can't place the arrow correctly.
I'm working on a reduced test case for this bug, but in the meantime we have to do without `-moz-padding-(start|end)`.